ownCloud/Settings/MoreSettingsSection.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  MoreSettingsSection.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 03/05/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2018, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+import WebKit
+import MessageUI
+
+// MARK: - Section identifier
+private let MoreSectionIdentifier: String = "settings-more-section"
+
+class MoreSettingsSection: SettingsSection {
+
+    // MARK: - More Settings Cells
+
+    private var helpRow: StaticTableViewRow?
+    private var sendFeedbackRow: StaticTableViewRow?
+    private var recommendRow: StaticTableViewRow?
+    private var privacyPolicyRow: StaticTableViewRow?
+
+    override init(userDefaults: UserDefaults) {
+        super.init(userDefaults: userDefaults)
+        self.headerTitle = "More".localized
+        self.footerTitle = "owncloud 2018 iOS beta"
+        self.identifier = MoreSectionIdentifier
+
+        createRows()
+        updateUI()
+    }
+
+    // MARK: - Creation of the rows.
+
+    private func createRows() {
+
+        helpRow = StaticTableViewRow(rowWithAction: { (_, _) in
+            let url = URL(string: "https://www.owncloud.com/help")
+            let title = "Help".localized
+            self.viewController?.navigationController?.pushViewController(WebViewController(url:url!, title: title), animated: true)
+        }, title: "Help".localized, accessoryType: .disclosureIndicator)
+
+        sendFeedbackRow = StaticTableViewRow(rowWithAction: { (_, _) in
+            self.sendMail(to: "apps@owncloud.com", subject: nil, message: nil)
+        }, title: "Send feedback".localized, accessoryType: .disclosureIndicator)
+
+        recommendRow = StaticTableViewRow(rowWithAction: { (_, _) in
+            let message = """
+                <p>I want to invite you to use ownCloud on your smartphone!</p>
+                <a href="https://itunes.apple.com/es/app/owncloud/id543672169?mt=8">Download here</a>
+                """
+            self.sendMail(to: nil, subject: "Try ownCloud on your smartphone!", message: message)
+        }, title: "Recommend to a friend".localized, accessoryType: .disclosureIndicator)
+
+        privacyPolicyRow = StaticTableViewRow(rowWithAction: { (_, _) in
+            let url = URL(string: "https://owncloud.org/privacy-policy/")
+            let title = "Privacy Policy".localized
+            self.viewController?.navigationController?.pushViewController(WebViewController(url:url!, title: title), animated: true)
+        }, title: "Privacy Policy".localized, accessoryType: .disclosureIndicator)
```

Comment:
I'm not sure this is the right privacy policy for the app. There's another privacy policy shown in the App Store (that can also be modified on a per-app basis, but needs to meet certain minimum requirements).

Also, opening that page in-app may be problematic under GDPR, as the user might not expect network access / a full browser session to occur there. If the app keeps showing a web page there, the user should probably be asked, if it's ok to show that page. (Disclaimer: I'm not a lawyer and this is not legal advice, just my two cents.)

We'll likely need some legal advice / legal guidance from ownCloud HQ to get this one right. @michaelstingl

Reply:
Should I prompt the user with an AlerView asking to launch the webview or leave it as it is for the moment?